test/test.js

```diff
@@ -0,0 +1,25 @@
+const assert = require('assert');
+const got = require('got');
+const cur = require('./../src/index.js');
+
+describe('return type check', () => {
+    const newInstance = new cur.MyServer(8000);
+
+    before(() => {
+        newInstance.startServer();
```

Comment:
Since `startServer` & `stopServer` are `async`, we should make hooks `async` too and use `await` to call these methods.